code/modules/materials/_materials.dm

```diff
@@ -441,6 +445,17 @@ INITIALIZE_IMMEDIATE(/obj/effect/gas_overlay)
 		return MAT_PHASE_LIQUID
 	return MAT_PHASE_SOLID
 
+// Returns the number of mols of material for the amount of solid or liquid units passed.
+/decl/material/proc/get_mols_from_units(units, phase)
```

Comment:
Not really, since gases have variable density, you can't tell how many moles you have from volume alone. Here we're just assuming the gases in the ice are in a liquid state, which doesn't make much sense but its the simplest thing to do right now.

Reply:
Warning added.